target/linux/ipq40xx/files/arch/arm/boot/dts/qcom-ipq4019-mf286d.dts

```diff
@@ -0,0 +1,462 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+// Copyright (c) 2022, Pawel Dembicki <paweldembicki@gmail.com>.
+
+#include "qcom-ipq4019.dtsi"
+#include <dt-bindings/soc/qcom,tcsr.h>
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+#include <dt-bindings/leds/common.h>
+
+/ {
+	model = "ZTE MF286D";
+	compatible = "zte,mf286d";
+
+	reserved-memory {
+		#address-cells = <0x01>;
+		#size-cells = <0x01>;
+		ranges;
+
+		rsvd1@87000000 {
+			reg = <0x87000000 0x500000>;
+			no-map;
+		};
+
+		wifi_dump@87500000 {
+			reg = <0x87500000 0x600000>;
+			no-map;
+		};
+
+		rsvd2@87b00000 {
+			reg = <0x87b00000 0x500000>;
+			no-map;
+		};
+	};
+
+	aliases {
+		led-boot = &led_internal;
+		led-failsafe = &led_internal;
+		led-running = &led_internal;
+		led-upgrade = &led_internal;
+	};
+
+	chosen {
+		bootargs-append = " root=/dev/ubiblock0_1";
+	};
+
+	gpio-export {
```

Comment:
Instead of gpio_export, I prefer the userspace-only [gpio_switch](https://github.com/openwrt/openwrt/blob/master/package/base-files/files/etc/init.d/gpio_switch).

This does have the advantage that the GPIO can be setup through (l)uci.
ipq40xx target already has the foundation file [/etc/board.d/03_gpio_switches](https://github.com/openwrt/openwrt/blob/master/target/linux/ipq40xx/base-files/etc/board.d/03_gpio_switches) file with some entries. Please add it there.

Reply:
Done.